make/mt-os/mt-stm32f4.mk

```diff
@@ -11,12 +11,16 @@ XI ?= $(XI_BINDIR)/libxively.a
 include make/mt-utils/mt-get-gnu-arm-toolchain.mk
 
 XI_STM32_PATH_SDK = $(HOME)/Downloads/xively-client-artifactory/st/STM32Cube_FW_F4_V1.13.0
+XI_STM32_PATH_SDK_ST = $(HOME)/Downloads/STM32F4xx_DSP_StdPeriph_Lib_V1.8.0/Libraries
```

Comment:
Will there eventually be a better place for this?  The Home Downloads directory is a bit...  I dunno. Temporary sounding.